src/frontend/packages/lib_components/src/data/sideEffects/refreshToken/index.ts

```diff
@@ -19,6 +20,12 @@ export const refreshToken = async (
   refreshToken: string,
   signal?: AbortSignal,
 ) => {
+  if (useJwt.getState().listRefreshJwtBlackListed.includes(refreshToken)) {
+    throw new Error('Refresh token is blacklisted');
```

Comment:
I'm not sure about this strategy:
- first we have the same problem we already had with multiple browser tab open
- There is a risk the request fails and is never made, leading to error in the application
I'm not really fan of this solution

Reply:
For the first point, it shouldn't be a problem, because the multiple calls are when you are located on 1 page (tab), so when you go to the second tab, the backend will have had the time to considered the old token as a blacklisted.

For the second point, I don't see how the request can never be made, because to be blacklisted it **must** have been already made.